_posts/2017-02-15-tensorflow-framework.markdown

```diff
@@ -0,0 +1,12 @@
+ ---
+ layout: post
+ title: tensorflow framework for neuroimaging
+ date: YYYY-MM-DD HH:MM
+ comments: true
+ external-url:
+ categories: tensorflow
+ ---
+
+We propose to develop a framework for using tensorflow in a neuroimaging setting including:
+  - Converting neuroimaging data (`.nii.gz` format) into a format uasable by Tensorflow
+  - Developing an example 3D convolutional neural network to label voxels
```

Comment:
I would suggest adding your name as the person interested in this.

`Interested parties: Paul Wighton`